frontend/src/scenes/authentication/loginLogic.ts

```diff
@@ -57,39 +65,48 @@ export const loginLogic = kea<loginLogicType>({
                 },
             },
         ],
-        authenticateResponse: [
-            null as AuthenticateResponseType | null,
-            {
-                authenticate: async ({ email, password }: { email: string; password: string }) => {
-                    try {
-                        await api.create('api/login', { email, password })
-                        return { success: true }
-                    } catch (e) {
-                        return {
-                            success: false,
-                            errorCode: (e as Record<string, any>).code,
-                            errorDetail: (e as Record<string, any>).detail,
-                        }
-                    }
-                },
+    })),
+
+    forms(({ actions }) => ({
+        login: {
+            defaults: {} as unknown as LoginForm,
```

Comment:
This will subtly break typing. `email` could be `undefined` sometimes now, without TS knowing. Any reason to not give blank defaults?

Reply:
Good point - will update